app/src/main/java/com/example/cinemapp/AppModule.kt

```diff
@@ -46,19 +51,24 @@ val appModule = module {
     single {
         MovieRepository(get(), get())
     }
+    single { URLMapper() }
+    single { SearchMapper(get()) }
+    single { HomeMapper(get()) }
+    single { DetailsMapper(get()) }
```

Comment:
What "details" are you trying to map? The following mapper name you did good -> `ActorDetailsMapper`, so do the same here :D